api/PclusterApiHandler.py

```diff
@@ -209,8 +209,11 @@ def get_cluster_config_text(cluster_name, region=None):
     else:
         info_resp = sigv4_request("GET", API_BASE_URL, url)
     if info_resp.status_code != 200:
-        print(info_resp.json())
-        return info_resp.json(), info_resp.status_code
+        if info_resp.status_code == 404:
```

Comment:
question: why do you still handle selectively the 404 case? Can you leave some comments for posterity?

Reply:
I'll remove this distinction and simplify the handling code